src/refdm/ingress.c

```diff
@@ -160,7 +160,7 @@ void *refdm_ingress_worker(void *arg)
             CACE_LOG_INFO("Got mif.recv result=%d, stopping", recv_res);
 
             // flush the input queue but keep the daemon running
```

Comment:
The REFDM doesn't use separate work threads for processing message contents, so this comment can be removed